### PR TITLE
Solve the palindrome checker puzzle

### PR DESCRIPTION
**Pull request by DGEs2018**
Could you please review my codes for this exercise ? Thank you !

### CODE REVIEW

File: challenges/is-it-palindrome.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+Write a method that takes a string and returns true if it is a palindrome. A palindrome is a string that is the same whether written backward or forward.
```
Comment (yhabib) [inline challenges/is-it-palindrome.js]: Description 🙌

File: challenges/is-it-palindrome.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Note: Don't forget that strings are case-sensitive.
+*/
+
+function palindrome(word) {
```
Comment (yhabib) [inline challenges/is-it-palindrome.js]: JSDoc 🙏 ?

File: challenges/is-it-palindrome.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	//to take case-sensitivity into consideration, convert the input to lowercase beforehand
+	let lowerCaseWord = word.toLowerCase();
```
Comment (yhabib) [inline challenges/is-it-palindrome.js]: Are you modifying the content of `lowerCaseWord`?? If not always prefer `const`. Rule of thumb, always define them as `const` and if you need to modify their content then refactor to let.

File: challenges/is-it-palindrome.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	// Define an empty string variable to use it collect characters after iteration
+	let characterHolder = '';
+	for (let i = word.length - 1; i >= 0; i--) {
+		// iterate backwards starting from the last index of the input
+		characterHolder = characterHolder + word[i];
+		//then concatenate each character one by one
+	}
+	//  now once the iteration is finished, compare the word composed, and if this is identical to the input word
+	if (lowerCaseWord === characterHolder) {
```
Comment (yhabib) [inline challenges/is-it-palindrome.js]: Two things:
1. Why do you compare to `lowerCaseWord`? Don't you want to compare to the original `word`?
2. In this case this `if/else` is unnecessary. The return value of your condition `lowerCaseWord === characterHolder` is going to implicitly return the `true` or `false` value that you explicitly return with the `if/else`. 🤓

Comment (DGEs2018) [reply challenges/is-it-palindrome.js]: True, even converting the input to lower case was only superfluous, as uppercase letter won't equal lower cases by no means. As for the, if I understand you well, `if/else`, I couldn't see any other way to print the Boolean true when the palindrome case is right. I could drop `else` but then would get `undefined`

**Pull request merged**